hack/make/release-deb

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -e
+
+# This script creates the apt repos for the .deb files generated by hack/make/build-deb
+#
+# The following can then be used as apt sources:
+# 	deb http://apt.dockerproject.org/repo $distro-$release $version
+#
+# For example:
+#	deb http://apt.dockerproject.org/repo ubuntu-trusy main
+#	deb http://apt.dockerproject.org/repo ubuntu-vivid testing
+#	deb http://apt.dockerproject.org/repo debian-wheezy experimental
+#	deb http://apt.dockerproject.org/repo debian-jessie main
+# 
+# ... and so on and so forth for the builds created by hack/make/build-deb
+
+: ${DOCKER_RELEASE_DIR:=$DEST}
+APTDIR=$DOCKER_RELEASE_DIR/deb
+
+# setup the apt repo (if it does not exist)
+mkdir -p "$APTDIR/conf" "$APTDIR/db"
+
+# create/update distributions file
+for suite in $(exec contrib/reprepro/suites.sh); do
+	
+	cat <<-EOF >> "$APTDIR/conf/distributions"
```

Comment:
We should probably drop the `>> "$APTDIR/conf/distributions"` here and just put `> "$APTDIR/conf/distributions"` on the `done` line of this for loop so that we overwrite the entire file wholesale each time:
```bash
for ...; do
    cat <<-EOF
      ....
    EOF
done > "$APTDIR/conf/distributions"
```